forge/config/features.js

```diff
@@ -1,5 +1,5 @@
 const featureFlags = [
-    { name: 'devices', default: false, isPublic: false }
+    { name: 'devices', default: true, isPublic: false }
```

Comment:
Tracing it back to api/index.js I understand the `isPublic` value doesn't influence anything for enabling this feature.

This line and the flagging should be removed in 0.7 though, I'll create a follow up issue.